pyiron_ontology/parser.py

```diff
@@ -165,3 +174,41 @@ def validate_values(graph):
                             (instance, on_property, some_values_from)
                         )
     return missing_triples
+
+
+def parse_workflow(
+    workflow,
+    NS,
+    graph=None,
+    inherit_properties=True,
+    hasNode=None,
+    hasSourceFunction=None,
+    hasUnits=None,
+    inheritsPropertiesFrom=None,
+):
+    if hasNode is None:
+        hasNode = NS.hasNode
+    if hasSourceFunction is None:
+        hasSourceFunction = NS.hasSourceFunction
+    if hasUnits is None:
+        hasUnits = NS.hasUnits
+    if inheritsPropertiesFrom is None:
+        inheritsPropertiesFrom = NS.inheritsPropertiesFrom
+    if graph is None:
+        graph = Graph()
+    workflow_label = NS[workflow.label]
+    graph.add((workflow_label, RDFS.label, Literal(workflow.label)))
+    for value in workflow.children.values():
```

Comment:
For me `for node in workflow:` is much more readable. The iterator on `pyiron_workflow.nodes.composite.Composite` makes the iteration equivalent, and I find the variable name change helpful since right after this we're doing things like `get_inputs_and_outputs(node)` which makes sense for `node`s and `function`s, but has no immediate connection to the more vague `value`.